app/routes/titiler/algorithms.py

```diff
@@ -1,21 +1,57 @@
-# from datetime import date
+from collections import OrderedDict
+from datetime import date
+
 import numpy as np
+from dateutil.relativedelta import relativedelta
 from pydantic import Field
 from rio_tiler.models import ImageData
 from titiler.core.algorithm import BaseAlgorithm
 
+from app.models.enumerators.alerts_confidence import DeforestationAlertConfidence
+
+START_DATE: str = "2014-12-31"  # start of record
+
+ALERT_CONFIDENCE_MAP: OrderedDict = OrderedDict(
+    {
+        DeforestationAlertConfidence.low: {
+            "confidence": 2,
+            "colors": {"red": 237, "green": 164, "blue": 194},
+        },
+        DeforestationAlertConfidence.high: {
+            "confidence": 3,
+            "colors": {"red": 220, "green": 102, "blue": 153},
+        },
+        DeforestationAlertConfidence.highest: {
+            "confidence": 4,
+            "colors": {"red": 201, "green": 42, "blue": 109},
+        },
+    }
+)
+
 
 class IntegratedAlerts(BaseAlgorithm):
     """Decode Integrated Alerts."""
 
     title: str = "Integrated Deforestation Alerts"
-    description: str = "Decode and vizualize alerts"
-
-    # START_DATE: np.datetime64 = np.datetime64("2014-12-31").astype(int)
+    description: str = "Decode and visualize alerts"
 
     # Parameters
-    start_date: int = Field(None, description="start date of alert")
-    end_date: int = Field(None, description="end date of alert")
+    default_start_date: str = (date.today() - relativedelta(days=180)).strftime(
```

Comment:
is there a specific reason you chose 6 months by default?

Reply:
Just because Flagship defaults to that, assuming they did some user research on that... If you ask me, I'd pick much shorter time frame to show near real-time alerts by default